.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/Common/ContentFilterSeverityResult.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.OpenAI;
+
+[CodeGenModel("AzureContentFilterSeverityResult")]
+public partial class ContentFilterSeverityResult
+{
+    [CodeGenMember("Severity")]
+    public ContentFilterSeverity Severity { get; }
+
+    [CodeGenMember("Filtered")]
+    public bool IsFiltered { get; }
```

Comment:
nit: I wonder if "Filtered" is appropriate enough. 

From the Framework Design Guidelines:
> For example, `CanRead` is more understandable than `Readable`. However, `Created` is actually more readable than `IsCreated`. Having the prefix is often too verbose and unnecessary, particularly in the face of IntelliSense in the code editors. It is just as clear to type `MyObject.Enabled =` and have IntelliSense give you the choice of true or false as it is to have `MyObject.IsEnabled =`, and the second approach is more verbose.

Reply:
I see the merits either way; Krzysztof was the one who suggested `IsFiltered` on ApiView:
![image](https://github.com/user-attachments/assets/18f4de36-ce0a-4a8d-a4ed-115932f4e07f)